_side.scss

```diff
@@ -304,4 +305,22 @@ a[href="/smallText"] {
 
 .side #search #searchexpando a {
 	color: $header-link-selected;
-}
\ No newline at end of file
+}
+
+.side .md a[href*="#game"]{
+	content:"";
+	display:inline-block;
+	height:50px;
+	width:50px;
+	background:url("%%blizz-icons%%") no-repeat;
+	margin-right:5px
+}
+
+.side .md a[href*="#game"]:hover{opacity:.8}
```

Comment:
Follow the in-place code style please. Missing a space between `hover` and `{` and there should be no inline declarations like this. This applies to the following lines as well.